backend/src/ee/services/license/__mocks__/license-fns.ts

```diff
@@ -26,8 +26,10 @@ export const getDefaultOnPremFeatures = () => {
     status: null,
     trial_end: null,
     has_used_trial: true,
-    secretApproval: false,
+    secretApproval: true,
     secretRotation: true,
     caCrl: false
   };
 };
+
+export const setupLicenceRequestWithStore = () => {};
```

Comment:
typo: You should do a CMD-F for `licence` and replace with `license`

Reply:
Yah i think i have to do that.

Reply:
Updated